src/PlaneWaveBasis.jl

```diff
@@ -229,10 +233,11 @@ treated explicitly. In this case all guess densities and potential
 functions must agree with the crystal symmetries or the result is
 undefined.
 """
-function PlaneWaveBasis(model::Model, Ecut::Number;
+function PlaneWaveBasis(model::Model,
+                        Ecut::Union{Number, Unitful.Quantity{T,D,Unitful.EnergyUnits}};
```

Comment:
Same here.

Reply:
I don't understand this one: it looks like the highest-level constructor for the plane wave basis and seems to be called explicitly in most of the examples. (I understand how the comment below about type specificity applies here.)